cmd/server-main_test.go

```diff
@@ -63,20 +63,66 @@ func TestGetListenIPs(t *testing.T) {
 	}
 }
 
+// Tests get host port.
+func TestGetHostPort(t *testing.T) {
+	testCases := []struct {
+		addr string
+		err  error
+	}{
+		// Test 1 - successful.
+		{
+			addr: ":" + getFreePort(),
+			err:  nil,
+		},
+		// Test 2 port empty.
+		{
+			addr: ":0",
+			err:  errors.New("Port cannot be empty or '0', please use `--address` to pick a specific port"),
```

Comment:
We could define but since they are not used elsewhere it's not defined in that manner, but not a bad idea either.